common/setups/rasr/gmm_system.py

```diff
@@ -826,6 +831,77 @@ def sat_recognition(
                     **kwargs,
                 )
 
+    # -------------------- output helpers  --------------------
+
+    def get_gmm_output(
+        self,
+        corpus_key: str,
+        corpus_type: str,
+        step_idx: int,
+        steps: RasrSteps,
+        extract_features: List,
+    ):
+        """
+        :param corpus_key: corpus name identifier
+        :param corpus_type: corpus used for: train, dev or test
+        :param step_idx: which iteration to take within one step
```

Comment:
Please clarify: "which iteration to take _within one step_" 
I would interpret it as e.g. select this iteration of align_split_acumulate_sequence of one specified GMM-Training-Step.

But below I see the code
```
steps.get_step_names_as_list()[step_idx - 1]
```
which reads to me like we select a specific step from the list of all defined steps.
(e.g. step_idx=1 -> monophone GMM; step_idx=3 -> triphone_sat GMM)

Which of these is true?